Gopkg.toml

```diff
@@ -1,6 +1,6 @@
 [[constraint]]
   name = "github.com/DataDog/agent-payload"
-  version = "4.15.0"
+  revision = "2994d48e214ea6e96528dddabfcb94bb600aa97a"
```

Comment:
this should be edited after https://github.com/DataDog/agent-payload/pull/30 is merged